test/functional/legacy/search_mbyte_spec.lua

```diff
@@ -0,0 +1,31 @@
+-- Test for search('multi-byte char', 'bce')
+
+local helpers = require('test.functional.helpers')
+local feed, insert, source = helpers.feed, helpers.insert, helpers.source
+local clear, execute, expect = helpers.clear, helpers.execute, helpers.expect
+
+describe('search_mbyte', function()
+  setup(clear)
```

Comment:
@justinmk I've updated `legacy2luatest.pl` to use `before_each` instead of `setup`.